app/views/profiles/_form.html.erb

```diff
@@ -18,6 +18,39 @@
                    label: "Nickname",
                    help: "Unique. Reserved for future use on menadevs.com" %>
 
+
+
+<label>Skills</label>
+<select id="skill_list" name="skill_list[]" class="skills-input" class="js-example-basic-single" multiple="multiple">
+
+  User.where(weekly_subscriber: true).find_each do |user|
+    NewsMailer.weekly(user).deliver_now
+  end
+
+
+  <% Tag.where(is_approved: true).find_each do |skill| %>
```

Comment:
Fixed, let me know if this is acceptable: https://github.com/mena-devs/tilde/pull/7/commits/94cc105244ec37d1adc72540686e23b7d9b343c2 (Note: the commit also includes bug fix)